### ISSUE TITLE
Names: the eternal quest

### ISSUE THREAD

**Issue by Kazuo256** [uspgamedev/backdoor#423]
We'll probably never be 100% satisfied with all names and terms we use, but let's try one more time.

## Primary Actions

Recently, we've been changing this to **Signature Ability** in the code. Any problems here?

## Widgets

A cool and cyberstuff-ish name for the permanent-type cards. I recently thought of "Components" too, but am not too happy either. Might be worth discussing.

## Background Attributes (currently Athletics, Arcana, and Mechanics)

Now this is the real problem.

I think Athletics is a good name, it works as "Athletics Attribute", "Athletics Skill", or "Athletics Level".

Arcana conflicts with the lore. Long story short, there are three magic systems, and only one of them is actually "Arcane Magic", but the Arcana attribute was intended for any "spellcasting" based background. Alternatives include "Artcasting" (which is more general than spellcasting) and "Caster" (as in "Caster-class Level"). The first one sounds good, but makes it look like "Art" cards are only meant for this background. The second would be very nice if we could find an "X-er" version for the other two attributes as well.

Mechanics is just plain weird. Alternatives include "Crafting" and "Gadgeteering".

**Comment by Kazuo256** [comment]
First suggestion:

In the lore, every kind of "special ability" or "skill" is called an "art". Even martial arts. So maybe we could extend the term "artcasting" like this:

+ Arcana -> Art-Cast
+ Mechanics -> Art-Craft
+ Athletics -> Art-Form (yeah, this is not too good)

To avoid repetition, we would just call the attributes "Cast Level", "Craft Level", "Form Level" or something.

Hopefully this leads to better ideas >.<

**Comment by Kazuo256** [comment]
> Athletics -> Art-Form (yeah, this is not too good)

After lurking around [Thesaurus](http://www.thesaurus.com/), I came up with "Art-Shift" for Athletics, since "shift" can mean a "move" or a "strategy". That would make for "shifting", "casting", and "crafting" skills, or, alternatively, "Shifter", "Caster", and "Crafter" class levels.

Still not satisfied though.

**Comment by Kazuo256** [comment]
Another option for "Mechanics" and "Crafting" is "Smith" (which can mean "craftsperson").

**Comment by OrenjiAkira** [comment]
Can you give a specific trait of "athletics" users? Like how do they usually battle? Do they have a common strategy? A common movement? We could use a metonym for it maybe

**Comment by Kazuo256** [comment]
There is no specific trait. Being athletic means training your body as your main tool for performing your job. Than *can* be fighting, but it could also be dancing, hunting, long-distance running, etc.

**Comment by OrenjiAkira** [comment]
A possible suggestion is inventing words for all classes.

**Comment by Kazuo256** [comment]
## New proposal

Instead of dividing in physical, magical and technological abilities, divide according to the Realms of Power in project:v's lore: corporis realm, arcane realm, and anima realm.

Thus, the attributes would be COR, ARC, and ANI. Or ATH, ARC, ANI. The associated colors are orange, purple, and green, respectivelly.

The downside of this approach is leaving "technology" out of the core attributes. To maintain the cyberpunk flavor, we would have to add "technology" as a mechanic in some other way.

**Comment by Kazuo256** [comment]
Bump, because we still need to think about the term "Widget".

As mentioned in the initial post, an alternative is "Component". Other suggestions?

**Comment by OrenjiAkira** [comment]
Despite being used to the name widget, "components" does sound more appropriate.
What does @rilifon think?

**Comment by rilifon** [comment]
I feel the same. Widgets nowadays automatically remind me of backdoor, but I guess components have more flavour. I don't mind if we make the switch

**Comment by Kazuo256** [comment]
Alright. Then, just to avoid changing our minds later a little, let's give this issue some more time in case a better name comes up. Otherwise, we change "widgets" to "components".

**Comment by rilifon** [comment]
How you guys are on this discussion? I think the one question left is "widget" or "components"?

@Detril so you can follow the discussion

**Comment by Kazuo256** [comment]
That's right. Beside, now we are considering implementing species and background traits as "indestructible widgets", so even "component" might not be a good name anymore.
